cql3/selection/abstract_function_selector.cc

```diff
@@ -79,6 +79,15 @@ abstract_function_selector::new_factory(shared_ptr<functions::function> fun, sha
             }
             return p->name().name == "countRows";
         }
+
+        virtual bool is_uda_reduce_selector_factory() const override {
+            auto p = dynamic_cast<functions::user_aggregate*>(_fun.get());
+            if (!p) {
+                return false;
+            }
+            return p->has_reducefunc();
+        }
```

Comment:
This should not be specific to UDA but to aggregates in general. SUM() and COUNT() can also have a reducer.

Reply:
Changed `has_reducefunc` to `is_reducible` and move this function from UDA to function class

Reply:
Please check that all build-in aggregate functions are now reducible. Otherwise, making them reducible later will require  a cluster feature again.

Reply:
Checked. All of them are reducible.